src/app/core/store/orders/orders.effects.ts

```diff
@@ -216,4 +231,24 @@ export class OrdersEffects {
     ofType(UserActionTypes.LogoutUser),
     mapTo(new ordersActions.ResetOrders())
   );
+
+  @Effect()
+  setOrderBreadcrumb$ = this.actions$.pipe(
+    ofRoute(),
+    mapToParam('orderId'),
+    whenTruthy(),
+    switchMapTo(this.store.pipe(select(getSelectedOrder))),
+    whenTruthy(),
+    debounceTime(0),
```

Comment:
why is this needed? if it is needed it is missing in the quoteRequestBreadcrumb effect

Reply:
the others are "features" and they are lazy loadet.
in this case we need it to be sure the translateservice works as expected